WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -133,6 +134,17 @@ private void StepConnectionConfirmationPhase()
 					var removedAliceCount = round.Alices.RemoveAll(x => alicesDidntConfirm.Contains(x));
 					round.LogInfo($"{removedAliceCount} alices removed because they didn't confirm.");
 
+					if (round.InputCount >= Config.MinInputCountByRound)
```

Comment:
If I'm looking correctly, then L148 represents `else` branch for L137. As such, it looks like conditions in L137-L156 can be simplified really.